packages/terra-show-hide/src/terra-dev-site/doc/example/CustomButtonTextShowHide.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import ShowHide from 'terra-show-hide/lib/ShowHide';
+
+const sentences = [];
+sentences.push(<p key="lorem1">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem2">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem3">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem4">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem5">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem6">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem7">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem8">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem9">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
+sentences.push(<p key="lorem10">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>);
```

Comment:
This could be defined during instantiation of the array.

```
    const sentences = [
      <p key="lorem1">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>,
      <p key="lorem1">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>,
      <p key="lorem1">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>,
    ];
```

Reply:
done: https://github.com/cerner/terra-core/pull/1956/commits/4ef5ed75a79ff0610b8a799e5033415112fc88d0